azure-local/manage/upgrade-sdn.md

```diff
@@ -0,0 +1,406 @@
+---
+title: Upgrade Infrastructure for Software Defined Networking (SDN) Managed by On-Premises Tools
+description: Learn how to upgrade infrastructure for SDN managed by on-premises tools.
+ms.topic: how-to
+ms.author: alkohli
+author: alkohli
+ms.date: 12/02/2025
+---
+
+# Upgrade infrastructure for Software Defined Networking managed by on-premises tools
+
```

Comment:
Should the applies to include Hyperconverged deployments of Azure Local running 2311.2 and later?